nexus-sdk-js/src/views/SparqlView/index.ts

```diff
@@ -0,0 +1,70 @@
+import { httpPost } from '../../utils/http';
+import { SparqlViewQueryException } from './exceptions';
+
+interface SparqlQuery {
+  content: string;
+}
+
+export interface SparqlViewResponse {
+  '@id': string;
+  '@type': string[];
+  _uuid: string;
+  _rev: number;
+  _deprecated: boolean;
+}
+
+interface SparqlViewQueryResponse {
```

Comment:
The format of the response is officially described in https://www.w3.org/TR/sparql11-results-json/ with the name "SPARQL Query Results JSON". Should we reflect that in the code, with a link to the spec in a comment as well?

It can also have no "results" but a "boolean" key with true or false as a value, but it needs to have one of the two. Also, the "head" member can have a "link" child, and also the individual results can have any additional key in addition to "type" and "value" (which are the only mandatory ones). Depending on the RDF type, they can have "datatype", or "xml:lang" or anything else who cannot know in advance.

Reply:
Awesome! Thanks, I'll update

Reply:
I've updated the interface and put a link to the RFC but for the name of the interface I'd rather keep what we have as it is consistent with the Elastic View and future views we can have d1b4646577174254e403296286f08e2db11d521e